go/cmd/vtctldclient/command/keyspaces.go

```diff
@@ -96,6 +96,16 @@ Otherwise, the keyspace must be empty (have no shards), or returns an error.`,
 		Args:                  cobra.ExactArgs(2),
 		RunE:                  commandSetKeyspaceServedFrom,
 	}
+	// SetKeyspaceDurabilityPolicy makes a SetKeyspaceDurabilityPolicy gRPC call to a vtcltd.
+	SetKeyspaceDurabilityPolicy = &cobra.Command{
+		Use:   "SetKeyspaceDurabilityPolicy [--durability-policy=policy_name] <keyspace name>",
+		Short: "Sets the durability-policy used by the specified keyspace.",
+		Long: "Sets the durability-policy used by the specified keyspace. Durability policy governs the durability of the keyspace by describing which tablets should be sending semi-sync acknowledgements to the primary. " +
```

Comment:
stylistically (sorry for not catching this in the first pass!), i would prefer using the `\`\`` style of quoting here and use actual newlines to separate. (for context, `Long` is only displayed when showing detailed help for a command, so it's actually preferred to write in a more man-page style).

you can see [`ApplySchema`](https://github.com/vitessio/vitess/blob/ccc8359755385c04b66599532f3274d505f538b1/go/cmd/vtctldclient/command/schema.go#L45-L57) for an example, though there are others as well

Reply:
Fixed this too. I have slightly improved the message as well

Reply:
![Screenshot 2022-05-13 at 9 18 22 PM](https://user-images.githubusercontent.com/35839558/168320334-faebc5dd-9876-4412-914e-8273ba670cda.png)